src/platforms/web/runtime/directives/model.js

```diff
@@ -35,9 +35,11 @@ export default {
       setSelected(el, binding, vnode.context)
       /* istanbul ignore if */
       if (isIE || isEdge) {
-        nextTick(() => {
+        let cb = () => {
```

Comment:
Note sure why CI is not running, but the linter setup will require using `const` here